app/frontend/components/pages/delegations/delegatePage.tsx

```diff
@@ -45,7 +96,7 @@ const StakePoolInfo = ({pool, gettingPoolInfo}) => {
             >
               <span className="delegation show-info">{''}</span>
             </a>
-            {`Tax: ${margin * 100 || ''}`}%
+            {`Tax: ${margin !== null ? margin * 100 : ''}`}%
```

Comment:
is margin guaranteed to not be undefined? If not, a safer comparison would be `margin != null` and also it should fallback to `n/a` or something like that, not an empty string

Reply:
I assume the reason why it falls back to `''` is so the field is empty, like other fields, when input is empty. But I agree we are mixing two different flows here. I will fix this.
![Screenshot_2021-02-15 AdaLite - Cardano Wallet](https://user-images.githubusercontent.com/69396388/107972600-612bf800-6fb4-11eb-8eef-6cf02454fb23.png)

Reply:
Fixed